minimal_example.py

```diff
@@ -69,6 +100,143 @@ async def main():
         )
         print(alice_pres_ex.json(by_alias=True, indent=2))
 
+        # Revoke credential v1
```

Comment:
Be sure to modify the function calls for the functions where we switched up the parameters